e2e/ui/README.md

```diff
@@ -64,3 +64,53 @@ runner. Typically you'll need to set `-bind` to your host's IP address
 [reverse proxy]: https://developer.hashicorp.com/nomad/tutorials/manage-clusters/reverse-proxy-ui
 [./input/proxy.nomad]: https://github.com/hashicorp/nomad/tree/main/e2e/ui/input/proxy.nomad
 [`NOMAD_*` environment variables]: https://developer.hashicorp.com/nomad/docs/commands#command-contexts
+
+## Contributing
+
+This guide contains documentation about contributing to the UI E2E tests using Playwright.
+
+If you're looking for information about using Playwright, please instead refer to the (Playwright documentation)[https://playwright.dev/docs/writing-tests].
+
+### Running the test suite
+
+In order to get the test suite running in an environment similar to what will happen in our CI pipeline, we'll be using the `./run.sh` bash script.
+
+This script, spins up a Docker container that will be used to run our Playwright scripts. The key note to keep in mind here is that, in local development, you'll be spinning up a Nomad server on your local machine and spinning up a container that will run our test scripts which means that you'll need to specify an address for the Nomad agent to bind for network services (in order to get access to the HTTP interface which allows us to use our `./api-client` to set-up and tear down testing arrangements).
```

Comment:
This is a bit of a run-on.

```suggestion
This script spins up a Docker container that will be used to run our Playwright
scripts. Keep in mind here is that, in local development, you'll be spinning up
a Nomad server on your local machine but spinning up a container to run our test
scripts. This means that you'll need to specify an address for the Nomad agent
to bind for network services (in order to get access to the HTTP interface which
allows us to use our `./api-client` to set-up and tear down testing
arrangements).
```

Also, as a style guide note: we hard-wrap markdown lines (your editor will probably have a shortcut for this). That makes it easier to review diffs on changes.